README.md

```diff
@@ -1,83 +1,87 @@
 
 
-Give me 1-2 weeks time for a reply. If you did not receive a reply at all, it
-might be a good idea to check your spam filter.
+Please allow 1-2 weeks for a reply. If you did not receive a reply at all, it might be a good idea to check your spam filter.
+
 
 ## License
 
-The extension is free and open source software, it is made available to you
-under the terms of the [GNU Affero General Public License (AGPLv3)](http://www.fsf.org/licensing/licenses/agpl-3.0.html).
+The [Sieve add-on](https://addons.thunderbird.net/addon/sieve/) is free and open source software, it is made available to you under the terms of the [GNU Affero General Public License (AGPLv3)](http://www.fsf.org/licensing/licenses/agpl-3.0.html).
 
 Refer to [Licensing information](https://github.com/thsmi/sieve/blob/master/LICENSING_INFO.md) for details.
 
+
 ## Releases
 
-You are looking for the most recent release?
+You are looking for the most recent [Sieve add-on](https://addons.thunderbird.net/addon/sieve/) release?
 
-Just go to [the releases page and download the latest stable xpi or view the release notes](https://github.com/thsmi/sieve/releases/latest).
+[Available by downloading the latest stable xpi at the Releases page (release notes can also be viewed)](https://github.com/thsmi/sieve/releases/latest).
 
-**Firefox user have to do a right click on the XPI download and select "Save link as" otherwise Firefox will steal the link and try to install the addon it into Firefox which will fail**
+**Firefox users: At the [ATN Sieve add-on page](https://addons.thunderbird.net/addon/sieve/), right-click on the “+ Download Now” button and select “Save Link As…”; otherwise Firefox try to install the add-on into Firefox which will fail.**
 
-The addon is restartless, so there is no need to restart Thunderbird upon installation and deinstallation.
+The add-on is restartless, so there is no need to restart Thunderbird upon installation and uninstallation.
 
-**Sometimes Thunderbird fails to invalidate its cache when updating a restartless addon. The result is the new version is displayed in the addons dialog but the old cached code is still be used. In such a case uninstall the addon, then restart thunderbird, restart it again and finally install it again. This ensures the cache gets correctly invalidated.** 
+**Sometimes Thunderbird fails to validate its cache when updating a restartless add-on. The result is the new version is displayed in the add-ons dialog but the old cached code is still used. In such cases, uninstall the add-on, then restart Thunderbird, restart it again and finally install it again. This ensures the cache gets correctly validated.** 
 
 ## Developments Builds
 
-You are looking for the latest "bleeding edge" features and willing to risk more instability?
+Are you looking for the latest “bleeding edge” features and willing to risk more instability?

```

Comment:
For HTML Mozilla's localization best practices suggest using the plain ASCII characters instead of the directional Unicode ones. Especially as those characters would be the only non 7bit ascii in this text file.

Reply:
> **Use Unicode characters over their ASCII counterparts when possible**
> 
> Strings should use directional quotation marks when possible.
> 
> * U+2018 and U+2019 (\u2018 and \u2019 in JavaScript) are the left and right single quotation marks, respectively. The right single quotation mark should be used as an apostrophe.
> * U+201C and U+201D (\u201c and \u201d in JavaScript) are the left and right double quotation marks, respectively. The pair should be used to surround statements and references to user input.
> o Example:  You have chosen a keyword that is currently in use by “%S”. Please select another.
> o Example: Search %1$S for “%2$S”
> 
> Non-directional single, and double, quotation marks should be used in reference to HTML code, or other languages where ASCII character usage is required, such as HTML. HTML does not prefer one over the other, and our codebase uses them interchangably as of March 2016.

https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices

Reply:
Not changed as Mozilla best practice recommends "Strings should use directional quotation marks when possible."